docs/specifications/tests/Makefile

```diff
@@ -31,8 +31,8 @@ delegation:
 	cat Delegation-TP/delegation*.md >> $(DOC)
 
 dnssec:
-	cat DNSSEC-TP/level.md > $(DOC)
-	cat DNSSEC-TP/dnssec*.md > $(DOC)
+	cat DNSSEC-TP/level.md >> $(DOC)
```

Comment:
With the new changes, level.md should be renamed as README.md

Reply:
@sandoche2k, that comes in a different PR.